scripts/generic.js

```diff
@@ -96,6 +109,55 @@ module.exports = robot => {
     res.send('.\n:bab-1::bab-2::bab-3:\n:bab-4::bab-5::bab-6:\n:bab-7::bab-8::bab-9:');
   });
 
+  // Listen to the Reminder message to count reactions
+  robot.hear(/Reminder: @badminton-(tuesday|thursday|saturday|testday) react with/i, res => {
+    resetReactions();
+    const messageId = res.message.id;
+    robot.hearReaction(res2 => {
```

Comment:
this listener doesn't get cleaned up; so does that mean that
1. `@badminton-tuesday react if you are playing`
2. `@badminton-thursday react if you are playing`
3. I react to the Tuesday message

bab will think that I'm playing Thursday?